tst/test-error/func-and-proc-call-trace.g

```diff
@@ -0,0 +1,96 @@
+informproc0 := function(l)
+Add(l,1);
+l();
+return; end;
+informproc0([]);
+quit;
```

Comment:
Perhaps insert `Print("\n\n");` after each `quit;`, to make the output a bit more readable?